topology/sof-glk-da7219.m4

```diff
@@ -102,31 +102,31 @@ DAI_ADD(sof/pipe-dai-capture.m4,
 	48, 1000, 0, 0)
 
 # capture DAI is DMIC0 using 2 periods
-# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+# Buffers use s32le format, with 48 frame per 1000us on core 0 with priority 0
 DAI_ADD(sof/pipe-dai-capture.m4,
 	4, DMIC, 0, dmic01,
 	PIPELINE_SINK_4, 2, s32le,
 	48, 1000, 0, 0)
 
 # playback DAI is iDisp1 using 2 periods
-# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+# Buffers use s32le format, with 48 frame per 1000us on core 0 with priority 0
 DAI_ADD(sof/pipe-dai-playback.m4,
         5, HDA, 3, iDisp1,
-        PIPELINE_SOURCE_5, 2, s16le,
+        PIPELINE_SOURCE_5, 2, s32le,
         48, 1000, 0, 0)
 
 # playback DAI is iDisp2 using 2 periods
-# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+# Buffers use s32le format, with 48 frame per 1000us on core 0 with priority 0
 DAI_ADD(sof/pipe-dai-playback.m4,
         6, HDA, 4, iDisp2,
-        PIPELINE_SOURCE_6, 2, s16le,
+        PIPELINE_SOURCE_6, 2, s32le,
         48, 1000, 0, 0)
 
 # playback DAI is iDisp3 using 2 periods
-# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+# Buffers use s32le format, with 48 frame per 1000us on core 0 with priority 0
```

Comment:
@keyonjie byw, would this be rejected by display driver or analog codec. The link DMA should still send either 24bit or 16 bit ???

Reply:
@lgirdwood I dont think so. the S32le and s16le are both acceptable for display driver, I think it will send 32bit directly when params format is S32. @mengdonglin correct me if wrong.

Reply:
@mengdonglin @keyonjie yes but what about analog codec ?